client/js/_author/components/common/tiny_wrapper.jsx

```diff
@@ -89,7 +90,11 @@ export default class TinyWrapper extends React.Component {
           id={`${this.id}-tinymce`}
           content={this.props.text}
           config={this.tinyMCEConfig()}
-          onBlur={(e) => { this.props.onBlur(e.target.getContent(), e.target.isDirty()); }}
+          onBlur={(e) => {
+            const content = e.target.getContent();
+            const isChanged = e.target.isDirty() || _.isEmpty(content);
```

Comment:
The onChange handler in tiny_wrapper is calling on blur. That is calling the blur function with a false is updated, and should probably call a change function instead. I think that may be where you are seeing calls to onBlur when the editor is empty and changed and isChanged is false.